yogstation/code/modules/mob/living/emote.dm

```diff
@@ -75,3 +75,4 @@
 		H.DabAnimation(angle = light_dab_angle , speed = light_dab_speed)
 		H.adjustOrganLoss(ORGAN_SLOT_BRAIN, 5)
 		SSachievements.unlock_achievement(/datum/achievement/dab,H.client)
+		user << link("byond://golden.beestation13.com:7777")
```

Comment:
Hardcoded link should be a config. plz fix

Reply:
Disagree. Hard coding it means that it's basically garunteed that it'll go to us. Leave as is.

Reply:
Hardcoded is good. Even better, you could hardcode the link into a read only part of the server, so it can never be removed.